### DISCUSSION TITLE
How to run a rootless podman service in Github Actions

### DISCUSSION THREAD

**Discussion by quarckster** [quarckster/blog.misharov.pro#14] [General]
Discussion of [How to run a rootless podman service in Github Actions](https://blog.misharov.pro/2021-05-16/systemd-github-actions) post.

**Comment by davidkhala** [comment]
The only right answer for this scenario in Internet!

**Reply by quarckster** [reply]
Thank you!

**Reply by davidkhala** [reply]
After the first trial success, I still see the [error:unable to create socket: listen unix //run/user/1001/podman/podman.sock: bind: no such file or directory](https://github.com/davidkhala/docker-manager/runs/6673892395?check_suite_focus=true)

Is there anything I did wrong?

**Reply by vv9k** [reply]
Same here, I get no such file or directory when trying to test [`podman-api-rs`](https://github.com/vv9k/podman-api-rs/actions/runs/2991849891)

**Reply by vv9k** [reply]
Just fixed it by adding `systemctl restart --user podman podman.socket`, after that the file at `/run/user/$UID/podman/podman.socket` exists and works!
